E2E-tests/config/substrate/staging_nodes.json

```diff
@@ -143,6 +143,10 @@
                 "lock": 817414141,
                 "send": 298945143
             }
-        }
+        },
+        "permissioned_min_ariadne_d_param_value": 6,
+        "permissioned_max_ariadne_d_param_value": 6,
+        "registered_min_ariadne_d_param_value": 4,
+        "registered_max_ariadne_d_param_value": 4
```

Comment:
maybe we can make it even simpler and skip the test if those are not provided at all?